pkg/model/components/kubelet.go

```diff
@@ -207,22 +184,11 @@ func (b *KubeletOptionsBuilder) BuildOptions(o interface{}) error {
 	if clusterSpec.Kubelet.FeatureGates == nil {
 		clusterSpec.Kubelet.FeatureGates = make(map[string]string)
 	}
-
-	if clusterSpec.CloudConfig != nil && clusterSpec.CloudConfig.AWSEBSCSIDriver != nil && fi.BoolValue(clusterSpec.CloudConfig.AWSEBSCSIDriver.Enabled) {
-		if _, found := clusterSpec.Kubelet.FeatureGates["CSIMigrationAWSComplete"]; !found {
-			clusterSpec.Kubelet.FeatureGates["CSIMigrationAWSComplete"] = "true"
-		}
-	}
 	if _, found := clusterSpec.Kubelet.FeatureGates["ExperimentalCriticalPodAnnotation"]; !found {
 		if b.IsKubernetesLT("1.16") {
 			clusterSpec.Kubelet.FeatureGates["ExperimentalCriticalPodAnnotation"] = "true"
 		}
 	}
 
-	// Set systemd as the default cgroup driver for kubelet from k8s 1.20
-	if b.IsKubernetesGTE("1.20") && clusterSpec.Kubelet.CgroupDriver == "" {
-		clusterSpec.Kubelet.CgroupDriver = "systemd"
-	}
-
```

Comment:
I think this was removed by mistake also.

Reply:
Yes, sorry. I will remove this changes

Reply:
Fixed.

Reply:
👍